tests/foreman/longrun/test_oscap.py

```diff
@@ -386,20 +471,68 @@ def test_positive_oscap_run_via_ansible_bz_1814988(
 
     :BZ: 1814988
     """
+    contenthost = rex_contenthost
+    os_version = contenthost.os_version.major
+    distro = f'rhel{os_version}'
+    result = contenthost.register(
+        module_org,
+        None,
+        ak_name[distro],
+        target_sat,
+        ignore_subman_errors=True,
+        force=True,
+        insecure=True,
+    )
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
+    rhel_repo = rhel_repos[distro]
+    profile = profiles[distro]
+    if distro not in ('rhel7'):
+        contenthost.create_custom_repos(**rhel_repo)
+    else:
+        contenthost.create_custom_repos(**{distro: rhel_repo})
```

Comment:
Isn't this the same?
```suggestion
        contenthost.create_custom_repos(distro: rhel_repo)
```